EidSuggest/Suggest.php

```diff
@@ -25,43 +25,37 @@
 
 # TSFE initialization
 
-tslib_eidtools::connectDB();
-$pageId     = filter_var(t3lib_div::_GET('id'), FILTER_SANITIZE_NUMBER_INT);
+$pageId     = filter_var(\TYPO3\CMS\Core\Utility\GeneralUtility::_GET('id'), FILTER_SANITIZE_NUMBER_INT);
 $languageId = filter_var(
-	t3lib_div::_GET('L'),
+	\TYPO3\CMS\Core\Utility\GeneralUtility::_GET('L'),
```

Comment:
Cannot use "use" here as it is and eID script not a class definition.
Or at least it was always avoided in core if it is only a file.